elfi/methods/post_processing.py

```diff
@@ -210,7 +210,7 @@ def _input_variables(self, model, sample, summary_names):
 
 
 def adjust_posterior(sample, model, summary_names, parameter_names=None, adjustment='linear'):
-    """Adjust the posterior using local regression.
+    r"""Adjust the posterior using local regression.
```

Comment:
Why does this line begin with "r"?

Reply:
Because of the `\` in the docstring Examples.  There doesn't seem to be any great solution for fixing linting AND the test, so I'll skip the particular tests as irrelevant.

Reply:
I'll stand corrected. There is. Fixing it now.